### PR TITLE
style system separation, clean up systems back to ECS

### PR DESCRIPTION
**Pull request by hanbollar**
what this entails:
- style system separation and back to main ECS
- ~~also https://github.com/Volumetrics-io/mrjs/issues/372~~
- misc:
  - ~~app --> App~~
  - moves old dom function from file --> mrjsUtils.CSS

### CODE REVIEW

**Comment by hanbollar** [comment]
need to do some remaining cleanup still for this pr

then next step would to be switch to an enum based update for the css sytem instead of all the separate methods - might do it in a separate pr instead depending on timing

**Comment by michaelthatsit** [comment]
As discussed on discord.

- mrjsUtils.Name -> mrjsUtils.name
- I'm seeing some z-fighting for UI that I'm not seeing on main
- The flickering bug returns after entering/exiting XR along with a big drop in FPS and eventual crash

**Comment by michaelthatsit** [comment]
z-fighting


https://github.com/Volumetrics-io/mrjs/assets/19628643/24098589-8f09-44fd-9ae1-f4bbb0199b14

**Comment by michaelthatsit** [comment]
FPS drop, flickering, crash

https://github.com/Volumetrics-io/mrjs/assets/19628643/7c0ad831-51e6-406b-870c-e57e7053ea40

**Comment by hanbollar** [comment]
naming fix handled by #380 , looking into the other noted things on this pr ~

**Comment by hanbollar** [comment]
> z-fighting
> 
>  z-fighting.mp4

michael said the z-fighting mightve been introduced by recent mr-panel changes, he's looking into ~

**Comment by hanbollar** [comment]
> FPS drop, flickering, crash
> 
>  flicker-crash.mp4

i'm still looking into this atm

File: src/core/entities/MRModel.js [hunk 1]
```diff
@@ -111,6 +111,7 @@ export class MRModel extends MRDivEntity {
         }
 
         this.loadModel();
+        console.log('loaded the mr-model');
```
Comment (hanbollar) [inline src/core/entities/MRModel.js:114]: need to remove

**Comment by hanbollar** [comment]
> given [#372 (comment)](https://github.com/Volumetrics-io/mrjs/issues/372#issuecomment-1935092880) - flickering and z-fighting are not specific to this pr - going to clean up some remaining items
> 
> if crashing is specific to this pr then will solve here, otherwise will solve that// z-fighting // layout (ie #372) in a separate pr

solved and merged in from main now that those are in -- going to do final checks here

**Comment by hanbollar** [comment]
made new bug for the crash as it seems very specific to debug mode - did enter/exit vr a few different times in a release session version and it was happy

**Pull request merged**